supervision/utils/video.py

```diff
@@ -117,6 +648,7 @@ def __exit__(self, exc_type, exc_value, exc_traceback):
         self.__writer.release()
 
 
+@DeprecationWarning
```

Comment:
No need to mark private classes / functions as deprecated. We do it only for classes / functions in public supervision API.